### PR TITLE
Encode the search in the page title and use consistent page titles fo…

### PR DESCRIPTION
**Pull request by cjcolvar**
…r increased context for screen readers

### CODE REVIEW

**Comment by cbeer** [comment]
Can this be merged now, or is it a 6.0 thing?

**Comment by cjcolvar** [comment]
I think this could be merged now and I'll have a followup 6.0-targeted PR to remove the deprecated i18n keys.

**Comment by mejackreed** [comment]
:+1: for :shipit: ... it potentially changes the default title behavior ... but we are about to cut a 6.0.0.pre release

File: app/helpers/blacklight/catalog_helper_behavior.rb [hunk 1]
```diff
@@ -273,4 +273,38 @@ def render_sms_action? config, options = {}
     !sms_mappings.blank?
   end
 
+  def render_search_to_page_title_filter(facet, values)
+    facet_config = facet_configuration_for_field(facet)
+    filter_label = facet_field_label(facet_config.key)
+    filter_value = case values.size
+    when 1
```
Comment (cbeer) [inline app/helpers/blacklight/catalog_helper_behavior.rb]: `Array#to_sentence` might be useful here?

File: app/views/saved_searches/index.html.erb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<% @page_title = t('blacklight.saved_searches.page_title', :application_name => application_name) %>
```
Comment (cbeer) [inline app/views/saved_searches/index.html.erb]: Can we try to use Ruby 1.9-style hashes?

File: app/helpers/blacklight/catalog_helper_behavior.rb (hunk 1 above)
Comment (cbeer) [inline app/helpers/blacklight/catalog_helper_behavior.rb]: Also, ruby 1.9-style hashes would be great.

**Comment by cjcolvar** [comment]
@cbeer how does that look?  Note for locales other than :en the ruby-i18n gem is necessary for the two_words_connector used by to_sentence otherwise it defaults to " and ".

**Comment by jrochkind** [comment]
I think there may already be I18n built into rails for the connectors? As used in http://apidock.com/rails/Array/to_sentence ?

**Pull request merged**